src/testHelpers/setupTests.js

```diff
@@ -33,15 +34,27 @@ console.error = (message, ...rest) => {
   if (didSuppressWarning(message)) return;
 
   if (REACT_ERRORS_REGEX.test(message)) {
+    TEST_SUITE_HAS_REACT_ERROR = true;
+    errors.push([message, ...rest, '\n\n']);
+  }
+
+  // error(message, ...rest);
```

Comment:
this will print out errors as they happen so I think this should be uncommented and the line with `const { error } = console;` should be added back in.